public/apps/account/test/__snapshots__/account-nav-button.test.tsx.snap

```diff
@@ -16,7 +16,7 @@ exports[`Account navigation button renders 1`] = `
     id="actionsMenu"
     isOpen={false}
     onClick={[Function]}
-    ownFocus={false}
+    ownFocus={true}
```

Comment:
maybe this was asked before, but why this change?

Reply:
this is because the unit test were failing due to jest snapshot mismatch... this is something worth the discussion.

Reply:
Note, this control was likely modified by a change in the styles from OpenSearch-Dashboards.  We should follow up and confirm that our UI looks clean and behaves as expected between 1.3.X and this 2.0.0 version.

I've created https://github.com/opensearch-project/security-dashboards-plugin/issues/938 for following up on this.